[IMP] Inventory: cluster picking (#11753)
odoo/documentation: content/applications/inventory_and_mrp/inventory/shipping_receiving/picking_methods/cluster.rst

.. |SO| replace:: :abbr:`SO (Sales Order)`
.. |SOS| replace:: :abbr:`SOs (Sales Orders)`

Cluster picking is an advanced picking method that combines the efficiency of :ref:`batch picking
<inventory/misc/batch_picking>` with immediate sorting during the picking process. It is best suited
for warehouses with high order volumes where organization and speed are critical.

Unlike batch picking, which requires a separate sorting step after picking, cluster picking sorts
items directly into designated bins or containers for each *sales order* (SO). This eliminates the
need for post-picking consolidation, making it ideal for operations prioritizing speed and accuracy.

> It is particularly effective in environments where immediate organization is crucial and orders

Reviewer: ```suggestion
Cluster picking is particularly effective in environments where immediate organization is crucial and orders
```

> contain a mix of items that need precise sorting during the picking process.

Reviewer: ```suggestion
contain a mix of items that need precise sorting during, rather than after, the picking process.
```

Reviewer: Optional suggestion to highlight cluster picking's differentiator from batch picking, as well as to emphasize why it's useful for users

Choose from the two options in the :guilabel:`Add to` field to either: add to :guilabel:`an existing
batch transfer`, or create :guilabel:`a new batch transfer`.

> Add a :guilabel:`Description` for this batch.

Reviewer: ```suggestion
Then, add a :guilabel:`Description` for this batch.
```

.. note::
   To find the package barcode, navigate to :menuselection:`Inventory app --> Products -->

>    Packages`, select the desired package, click the :icon:`fa-cog` :guilabel:`(gear)` icon at the

Reviewer 2: Actually, I have another thought! 
Since the Barcode scope is missing a doc about cluster and wave pickings anyway (the only existing one is [this outdated one](https://www.odoo.com/documentation/18.0/applications/inventory_and_mrp/barcode/operations/transfers_scratch.html) from Odoo 15 or 16), you can alternatively rework that barcode doc for all of the picking methods (5 points), and simply remove the barcode section from all of the docs about picking methods in Inventory!

>    Packages`, select the desired package, click the :icon:`fa-cog` :guilabel:`(gear)` icon at the

Reviewer 2: Since we're on the topic about more doc ideas, if you're feeling up for it, Jess, you can also write a comparison doc here:
![image](https://github.com/user-attachments/assets/f2abb54c-2f00-4596-8be6-59e81193a89c)

Comparing the pros and cons of all the picking methods, with links to each.

>    Packages`, select the desired package, click the :icon:`fa-cog` :guilabel:`(gear)` icon at the

Author: As I mentioned in the batch picking doc, I think it is best if I remove this section from each of these docs and focus on creating a separate doc to cover this information.
